streaming/transducer.scrbl

```diff
@@ -106,13 +107,20 @@ early, before the input sequence is fully consumed.
 
 @defproc[(append-mapping [f (-> any/c sequence?)]) transducer?]{
  Constructs a @tech{transducer} that applies @racket[f] to input elements and
- emits each element in the returned sequence downstream.
+ emits @italic{each} element in the returned sequence downstream.
+ 
+ This is similar to Java's
+ @hyperlink["https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html#flatMap-java.util.function.Function-"]{@tt{Stream.flatMap}}
+ method.
 
  @(examples
    #:eval (make-evaluator) #:once
    (transduce (set 'red 'green 'blue)
               (append-mapping symbol->immutable-string)
-              #:into into-string))}
+              #:into into-string))
+   (transduce (list 3 1 2)
+              (append-mapping (lambda (n) (in-range n))
```

Comment:
```suggestion
              (append-mapping (lambda (n) (in-range 0 n))
```
nit: make it explicit that this is a range from 0 to n (I find the way `in-range` behaves with only one argument kind of confusing, and it doesn't seem worth it given that it only saves two characters over explicitly using 0 for the start.)